dist/docker/docker-entrypoint.py

```diff
@@ -11,7 +11,7 @@
 from scylla_util import sysconfig_parser
 
 def extract_args_from_conf(arguments):
-    scylla_server_conf = sysconfig_parser('/etc/default/scylla-server')
+    scylla_server_conf = sysconfig_parser('/etc/sysconfig/scylla-server')
```

Comment:
Let's make this more robust by checking for both, so we can create a debian container if we want.